src/simplewallet/simplewallet.cpp

```diff
@@ -2500,6 +2545,28 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
 
     switch (transfer_type)
     {
+      case TransferStake:
+        bc_height = get_daemon_blockchain_height(err);
+        if (!err.empty())
+        {
+          fail_msg_writer() << tr("failed to get blockchain height: ") << err;
+          return true;
+        }
+        unlock_block = bc_height + locked_blocks;
+        ptx_vector = m_wallet->create_transactions_2(dsts, fake_outs_count, unlock_block /* unlock_time */, priority, extra, m_trusted_daemon);        
+
+        for (tools::wallet2::pending_tx& tx : ptx_vector)
```

Comment:
Allowing a vector of >1 transactions here is likely to complicate things on the supernode side: you'll end up with multiple stake transactions with different stake amounts, and the transactions will likely not even make it into the same block.  It's possible to make that work, but as I said, more complicated: it might be easier just to fail here with an error if the outputs can't fit into a single transaction.

Reply:
You are right. Will rework it with error reporting for >1 transactions.

Reply:
This code was removed.